packages/fxa-auth-server/lib/senders/emails/fluentLocalizer.ts

```diff
@@ -0,0 +1,97 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { DOMLocalization } from '@fluent/dom';
+import { FluentBundle, FluentResource } from '@fluent/bundle';
+import { negotiateLanguages } from '@fluent/langneg';
+import { JSDOM } from 'jsdom';
+import * as fs from 'fs';
+import path from 'path';
+import { renderWithOptionalLayout, context } from './renderer';
+import availableLocales from 'fxa-shared/l10n/supportedLanguages.json';
+
+const OTHER_EN_LOCALES = ['en-NZ', 'en-SG', 'en-MY'];
+
+class FluentLocalizer {
+  constructor() {}
+
+  async localizeEmail(
+    templateName: string,
+    layoutName: string,
+    mailSubject: string,
+    variables: Record<any, any>,
+    acceptLanguage: string
+  ) {
+    const htmlDocument = renderWithOptionalLayout(
+      templateName,
+      { ...variables, ...context },
```

Comment:
The templateValues object could be passed down in a better way. Maybe this can be done as part of the cadReminder ticket